README.md

```diff
@@ -2,25 +2,37 @@
 
 [![CircleCI](https://circleci.com/gh/fellowapp/prosemirror-py.svg?style=shield)](https://circleci.com/gh/fellowapp/prosemirror-py)
 [![Code Coverage](https://codecov.io/gh/fellowapp/prosemirror-py/branch/master/graph/badge.svg?style=flat)](https://codecov.io/gh/fellowapp/prosemirror-py)
-[![Python Version](https://img.shields.io/pypi/pyversions/prosemirror-py.svg?style=flat)](https://pypi.org/project/prosemirror-py/)
-[![PyPI Package](https://img.shields.io/pypi/v/prosemirror-py.svg?style=flat)](https://pypi.org/project/prosemirror-py/)
-[![License](https://img.shields.io/pypi/l/prosemirror-py.svg?style=flat)](https://github.com/fellowapp/prosemirror-py/blob/master/LICENSE.md)
+[![Python Version](https://img.shields.io/pypi/pyversions/prosemirror.svg?style=flat)](https://pypi.org/project/prosemirror-py/)
```

Comment:
These shields are still linking to `pypi.org/project/prosemirror-py/`, presumably they oughta be updated as well?